v2/src/modules/block/components/MoreBlockDetails.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+    <v-card variant="flat" class="pa-4 pa-sm-6">
+        <template v-if="props.isLoading || !props.blockDetails">
+            <v-row class="my-2">
+                <v-col cols="12" md="6">
+                    <div class="skeleton-box rounded-xl mt-1" style="height: 24px"></div>
+                </v-col>
+            </v-row>
+            <div v-for="i in 5" :key="i" class="skeleton-box rounded-xl my-5" style="height: 24px"></div>
+        </template>
+        <template v-else>
+            <v-row>
```

Comment:
Lets actually put all fo this column on a new line, otherwise right now values do not fit.